ext/cjson/Kconfig

```diff
@@ -8,6 +8,7 @@
 
 config  CJSON_LIB
 	bool "cJSON library"
+	depends on NEWLIB_LIBC
```

Comment:
This will increase the size of the binary, are you sure this is what you want?

Reply:
As stated in the PR text

`Forced to use newlibc as sscanf and strtod does not exsists in minimal libc`

But is there a way to get around this ? Maybe just copy those functions.

Reply:
Well it should have been there anyways. Since cJSON uses `sscanf` and `strtod` which doesn't seem to exsist in `minimal libc`. I don't know if there is a configuration option which could enable this?

I haven't found an option/config for removing the functionality of printing/parsing double numbers from cJSON so it can't easily be removed from the library.